python/cudf/cudf/core/accessors/string.py

```diff
@@ -532,25 +535,6 @@ def join(
 
         return self._return_or_inplace(data)
 
-    def _split_by_character(self) -> ListColumn:
-        col = self._column.fillna("")  # sanitize nulls
-        result_col = col.character_tokenize()
-        if isinstance(result_col.dtype, ListDtype) and (result_col.size > 0):
-            return result_col  # type: ignore
-
-        offset_col = col.children[0]
-        child_col = result_col.children[1]
-
-        return ListColumn(
-            data=None,
-            size=len(col),
-            dtype=ListDtype(col.dtype),
-            mask=col.mask,
-            offset=0,
-            null_count=0,
-            children=(offset_col, child_col),  # type: ignore[arg-type]
-        )
-
```

Comment:
Why can we drop this? Do we not need a call to copy or something after the `character_tokenize` call that was inlined on line 505? Are these deleted lines all really just redundant at this point?